src/app/mail/mail-detail/mail-detail.component.html

```diff
@@ -159,9 +154,17 @@ <h6 class="text-dark font-weight-bold mb-0">{{childMail.sender}}</h6>
                       </td>
                     </tr>
                     <tr *ngIf="decryptedContents[childMail.id]">
-                      <td id="{{childMail.id}}-mail-content">
+                      <td id="{{childMail.id}}-mail-content" [ngClass]="{'show_gmail_extra': showGmailExtraContent}">
```

Comment:
`showGmailExtraContent ` shouldn't be a single variable, it should take into account the index of email reply, so that it only opens or close the respective reply extra contant.

like :: `showGmailExtraContent[i]`